zha/zigbee/cluster_handlers/measurement.py

```diff
@@ -202,3 +203,15 @@ class FormaldehydeConcentrationClusterHandler(ClusterHandler):
             config=(REPORT_CONFIG_MIN_INT, REPORT_CONFIG_MAX_INT, 0.000001),
         ),
     )
+
+
+@registries.CLUSTER_HANDLER_REGISTRY.register(WindSpeedMeasurement.cluster_id)
+class WindSpeedMeasurementClusterHandler(ClusterHandler):
+    """Wind Speed measurement cluster handler."""
+
+    REPORT_CONFIG = (
+        AttrReportConfig(
+            attr=WindSpeedMeasurement.AttributeDefs.measured_value.name,
+            config=(REPORT_CONFIG_MIN_INT, REPORT_CONFIG_MAX_INT, 0.01),
```

Comment:
In the ZCL, `MeasuredValue = 100 x Wind Speed in m/s`. This sets the minimum reportable change to 1 m/s, correct?

Reply:
<img width="793" alt="Screenshot 2025-03-24 at 15 03 47" src="https://github.com/user-attachments/assets/16cec0d5-6321-4249-ba02-9c4f055725f4" />

It should be the same as temperature. In app its float with 2 decimals -> `0.01` so that is a minimal change.
But the attribute value is incremented x 100 so save it as uint16.

Reply:
I think it's fine, just double checking. I don't think any devices exist that require this default and we can change it if they do.